end-to-end-test/shared/specUtils.js

```diff
@@ -189,6 +189,22 @@ function setDropdownOpen(
     );
 }
 
+/**
+ * @param {string} url
+ * @returns {string} modifiedUrl
+ */
+function getUrl(url) {
+    if (!useExternalFrontend) {
+        console.log('Connecting to: ' + url);
+    } else {
+        const urlparam = useLocalDist ? 'localdist' : 'localdev';
```

Comment:
where does useLocalDist come from?

Reply:
That is probably a legacy env var. I extracted this logic from goToUrlAndSetLocalStorage fn